client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -734,205 +815,226 @@ export const ApplicationsTable: React.FC = () => {
             activeAppInDetailDrawer ? getTask(activeAppInDetailDrawer) : null
           }
         />
-      </ConditionalRender>
-      <Modal
-        title={
-          createUpdateApplications
-            ? t("dialog.title.updateApplication")
-            : t("dialog.title.newApplication")
-        }
-        variant="medium"
-        isOpen={isCreateUpdateApplicationsModalOpen}
-        onClose={() => setSavesApplicationsModalState(null)}
-      >
-        <ApplicationForm
-          application={createUpdateApplications}
+        <Modal
+          title={
+            createUpdateApplications
+              ? t("dialog.title.updateApplication")
+              : t("dialog.title.newApplication")
+          }
+          variant="medium"
+          isOpen={isCreateUpdateApplicationsModalOpen}
           onClose={() => setSavesApplicationsModalState(null)}
-        />
-      </Modal>
-      <Modal
-        isOpen={isCopyAssessmentModalOpen}
-        variant="large"
-        title={t("dialog.title.copyApplicationAssessmentFrom", {
-          what: applicationToCopyAssessmentFrom?.name,
-        })}
-        onClose={() => {
-          setAapplicationToCopyAssessmentFrom(null);
-          fetchApplications();
-        }}
-      >
-        {applicationToCopyAssessmentFrom && (
-          <BulkCopyAssessmentReviewForm
-            application={applicationToCopyAssessmentFrom}
-            assessment={
-              getApplicationAssessment(applicationToCopyAssessmentFrom.id)!
-            }
-            isSubmittingBulkCopy={isSubmittingBulkCopy}
-            setIsSubmittingBulkCopy={setIsSubmittingBulkCopy}
-            isCopying={isCopying}
-            createCopy={createCopy}
-            onSaved={() => {
-              setAapplicationToCopyAssessmentFrom(null);
-              fetchApplications();
-            }}
+        >
+          <ApplicationForm
+            application={createUpdateApplications}
+            onClose={() => setSavesApplicationsModalState(null)}
```

Comment:
There's a preexisting typo here that we could correct, `setSavesApplicationModalState` -> `setSaveApplicationModalState` if you want